.github/workflows/industrial_ci_action.yml

```
@@ -0,0 +1,27 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on: [push, pull_request]
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
```

Add the Noetic build instead of Melodic testing. The build result for this msg package shouldn't differ between main and testing, because we rely on standard msgs.
```suggestion
          - {ROS_DISTRO: melodic, ROS_REPO: main}
          - {ROS_DISTRO: noetic, ROS_REPO: main}
```

---

took the change and then rebased it into the first commit